src/index.ts

```diff
@@ -31,62 +10,79 @@ const Error = Variant({
  * - `insert`, `get` and `remove` operations have a constant time complexity - O(1)
  * - data stored in the map survives canister upgrades unlike using HashMap where data is stored in the heap and it's lost after the canister is upgraded
  * 
- * Brakedown of the `StableBTreeMap(text, Message)` datastructure:
+ * Brakedown of the `StableBTreeMap(string, Message)` datastructure:
  * - the key of map is a `messageId`
  * - the value in this map is a message itself `Message` that is related to a given key (`messageId`)
  * 
  * Constructor values:
- * 1) text - the type of the key in the map
- * 2) Message - the type of the value in the map.
- * 3) 0 - memory id where to initialize a map.
+ * 1) 0 - memory id where to initialize a map.
  */
-const messagesStorage = StableBTreeMap(text, Message, 0);
 
-export default Canister({
-    getMessages: query([], Result(Vec(Message), Error), () => {
-        return Ok(messagesStorage.values());
-    }),
-    getMessage: query([text], Result(Message, Error), (id) => {
-        const messageOpt = messagesStorage.get(id);
+/**
+    This type represents a message that can be listed on a board.
+*/
+class Message {
+    id: string;
+    title: string;
+    body: string;
+    attachmentURL: string;
+    createdAt: Date;
+    updatedAt: Date | null
+}
+
+const messagesStorage = StableBTreeMap<string, Message>(0);
```

Comment:
I would recommend declaring this variable with let to indicate mutability, just a subjective best practice I have adopted